.github/workflows/spdx_license_bot.yaml

```diff
@@ -0,0 +1,64 @@
+name: spdx_license_bot
+
+on:
+  # This should ideally trigger whenever there is a commit to the [SPDX License repository](https://github.com/spdx/license-list-data).
+  # However, this is not yet possible see: https://github.com/orgs/community/discussions/26323
+  schedule:
+    # At 08:04 on every day-of-week from Monday through Friday.
+    - cron: "4 8 * * 1-5"
+  workflow_dispatch:
+
+env:
+  # The code Mason returns when there are changes detected after generation, see:
+  # https://github.com/felangel/mason/tree/master/packages/mason_cli#detecting-changes-after-generation
+  MASON_DETECTED_CHANGES_CODE: 70
+
+jobs:
+  build:
+    defaults:
+      run:
+        working-directory: tool/spdx_license
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: 📚 Git Checkout
+        uses: actions/checkout@v4
+
+      - name: 🎯 Setup Dart
+        uses: dart-lang/setup-dart@v1
+        with:
+          sdk: 3.1.0
```

Comment:
I would just go with stable and it will break eventually, that way we always use whatever is stable instead of old :P